collect_app/src/main/java/org/odk/collect/android/activities/GeoPointMapActivity.java

```diff
@@ -366,7 +366,7 @@ private void upMyLocationOverlayLayers() {
     }
 
     private void overlayMyLocationLayers() {
-        if (draggable & !readOnly) {
+        if (draggable && !readOnly) {
```

Comment:
I’ve never seen anybody use bitwise `and` on a boolean, much less a capital-B Boolean. Surely that can’t be intentional. And why is draggable not a primitive? Further exploration may be warranted.